appeals-service-api/src/__tests__/appeals.spec.js

```diff
@@ -61,7 +58,7 @@ describe('Appeals API', () => {
     expect(response.statusCode).toBe(404);
   });
 
-  test('PUT /api/v1/appeals/{id} - It responds with an updated appeal', async () => {
+  test.skip('PUT /api/v1/appeals/{id} - It responds with an updated appeal', async () => {
```

Comment:
@frank-dodion this test is failing but it seems that it's doing what it's supposed to do. Update based with the change made by the service. Can you confirm please ?

<img width="1158" alt="Capture d’écran 2020-12-10 à 14 57 08" src="https://user-images.githubusercontent.com/3695996/101781358-fbc11400-3af7-11eb-8311-6a1bf293ce17.png">

Reply:
Let me see..

Reply:
It seems to pass for me locally. I typically use npm run test:tdd from the appeals-service-api folder to get rapid feeddback.

Reply:
Is it failing in the ci checks?

Reply:
Not sure as I fixed something else, I push without the skip to see. Thanks

<img width="693" alt="Capture d’écran 2020-12-10 à 15 25 16" src="https://user-images.githubusercontent.com/3695996/101784524-e8b04300-3afb-11eb-840c-5e33b3b26b8d.png">

It's this part which somehow change the date on my laptop and top yours

Reply:
The ci passed, that's weird.

Reply:
For most of the tests we generate a new appeal and set the decision date to yesterday. If could be any past date but I wanted it to stay in the valid period so it could not be a fixed date. I suppose it could be a time zone thing because the essentially we strip out the first ten characters to get a string like this YYYY-MM-DD from an ISO date string.